### ISSUE TITLE
Changing dashboard JSON does not update until after restart

### ISSUE THREAD

**Issue by torarnv** [vudash/vudash#66]
There doesn't seem to be any hot-reload of the dashboards. Changing e.g. the width or height of a widget.

I don't know if this limitation also applies to widgets. Coming from Dashing/Smashing, the ability to quickly change a widget was very handy.

**Comment by antony** [comment]
Yeah - there is no hot-reloading currently, as I haven't had the opportunity to implement it. I did add a feature to add a dashboard dynamically, but not to edit an existing one.
